week_1/5_apr/layout.css

```diff
@@ -0,0 +1,76 @@
+body {
+    height: 3000px;
+}
+
+.fixed {
+    position: fixed;
+    background-color: red;
+    width: 100%;
+    text-align: center;
+}
+
+.fixed .h1-fixed {
+    border: 8px solid wheat;
+    display: inline-block;
+    padding: 8px 12px;
+}
+
+.static {
+    position: static;
+}
+
+.relative {
+    position: relative;
```

Comment:
@codal-kcolaco What is position and what are it's type and use of it's type?

Reply:
@abhi-codal position helps us to define how an element is positioned within its containing element. The main types of position are:

1. **Static**: Elements with static flow are positioned according to normal flow of document
2. **Relative**: These elements are positioned relative to their normal position. We can move them using top, bottom, right and left properties. But they still occupy the same space as before.
3. **Absolute**: These are positioned relative to a positioned element. They don't affect the layout of other elements and do not occupy the same space as before.
4. **Sticky**: These elements are positioned based on user's scroll position. They behave like a relative position element until they reach a specified scroll position where they become fixed.
5. **Fixed**: These elements are positioned relative to the browser window. They do not move when the page is scrolled.